manifester/settings.py

```diff
@@ -20,6 +28,7 @@
 settings = Dynaconf(
     settings_file=str(settings_path.absolute()),
     ENVVAR_PREFIX_FOR_DYNACONF="MANIFESTER",
+    preload=f"{str(FRAMEWORK_DIRECTORY)}/*.yaml",
```

Comment:
I'll need to look into this more to be sure, but my first thought is no - assuming by `the framework config` you mean `the settings object being used by the framework during runtime`. Manifester will read the framework config files and load the values into its own dynaconf instance, but that scope should be distinct from e.g. Robottelo's dynaconf instance. I opened this PR because I found that Manifester was unable to access Robottelo's settings when used in tests. If it were possible for Manifester to override the Robottelo settings being used by Robottelo, I think Manifester would have been able to access Robottelo's settings without the changes in this PR. But please let me know if my thinking here seems flawed.

If by `the framework config` you mean `the framework settings being used in Manifester's settings object during runtime`, then the answer is probably yes, though I don't believe that would happen without duplicating framework settings in Manifester.

Reply:
So I think that's what I was trying to address with my second paragraph. I don't see how config values set by Robottelo could be overwritten by the `settings_file` unless settings in Robottelo were duplicated in Manifester. There are currently no shared settings between the two, and I think it should be easy enough to avoid duplicating settings in the future. 

I also don't see a way around loading Robottelo's settings first. In order for Manifester to access a value from Robottelo's settings for its own settings (which this PR is designed to enable), Manifester will need to load Robottelo's settings before its own settings.

Does this answer your question?

Reply:
Lots of 'offline' discussion went on concerning this pattern and PR content.

The updates to config have been removed, and the intended pattern is as follows:

1. `Manifester(manifest_category='golden_ticket')`  > Will read from manifester_settings.manifest_category for manifest data
2. `Manifester(manifest_category=dict(subscription_data=[dict(name="some_subscription", quantity=100)], simple_content_access=True)`  > Will not read from `manifester_settings`, and will operate on the passed subscription/manifest information.